### PR TITLE
Session in Redis

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@ django-reversion==1.8.6
 django-taggit==0.17.1
 django-taggit-serializer==0.1.5
 djangorestframework==3.2.2
+django-redis-sessions==0.5.0
```
Comment (mkurek) [inline requirements/base.txt]: this should be IMO in prod.txt

File: src/ralph/settings/base.py [hunk 2]
```diff
@@ -180,6 +180,13 @@
     'DB': int(os.environ.get('REDIS_PORT_6379_TCP_DB', 0)),
 }
 
+SESSION_ENGINE = 'redis_sessions.session'
```
Comment (mkurek) [inline src/ralph/settings/base.py]: ditto - in `prod.py` settings. @ar4s what do you think?

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**